config/defaults.js

```diff
@@ -1,5 +1,16 @@
-export default {
+const commonjs = require('rollup-plugin-commonjs'),
+  npm = require('rollup-plugin-npm'),
```

Comment:
Now we got to move to `rollup-plugin-node-resolve` :-)
They've changed the name of that plugin.
https://github.com/rollup/rollup-plugin-node-resolve